### PR TITLE
add test cases

### PR DESCRIPTION
**Pull request by Doplgangr**
New test cases page for collaborating on test cases

### CODE REVIEW

File: _guide/08-test-cases.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ 
+Restrict vault opening 
+ 
+**Click on the newly created vault and enter CORRECT password** 
+ 
+Open the vault 
+ 
+## File Adding and Handling 
+ 
+**Add few files (Photos)**
```
Comment (L-Henke) [inline _guide/08-test-cases.md:70]: Do you mean **n**ew files?

Comment (Doplgangr) [inline _guide/08-test-cases.md:70]: No, it literally meant adding a few files. Maybe I should have made it
clearer.

**Comment by L-Henke** [comment]
Should be OK for now. I'll add some test cases later.

**Comment by Doplgangr** [comment]
I'm not entirely sure but the page is not showing up. I'll check to see what's wrong

**Pull request merged**